### PR TITLE
WpOrgApi: allow specifying fields request param

### PR DESCRIPTION
**Pull request by swissspidy**
Required for wp-cli/extension-command#394

### CODE REVIEW

**Comment by swissspidy** [comment]
@wp-cli/committers would appreciate a review here to unblock https://github.com/wp-cli/extension-command/pull/393.

The test failures are unrelated, see #5892

**Review by danielbachhuber — APPROVED** [review]

**Pull request merged**
